Undeprecate v2 metrics (#1375)
minio/docs: source/operations/monitoring/healthcheck-probe.rst

online and functional. Any other HTTP codes indicate an issue with reaching
the server, such as a transient network issue or potential downtime.

> The healthcheck probe alone cannot determine if a MinIO server is offline - only that the current host machine cannot reach the server.

Reviewer: ```suggestion
The healthcheck probe alone cannot determine if a MinIO server is offline.
Instead, the probe determines whether the current host machine can reach the server.
```

> Consider configuring a Prometheus :ref:`alert <minio-metrics-and-alerts>` using the :ref:`metrics v3 <minio-metrics-and-alerts>` ``minio_cluster_health_nodes_offline_count`` or :ref:`v2 <minio-metrics-v2>` ``minio_cluster_nodes_offline_total`` metrics to detect whether one or more MinIO nodes are offline.

Author: This seems to be the equivalent v3 metric

> Consider configuring a Prometheus :ref:`alert <minio-metrics-and-alerts>` using the :ref:`metrics v3 <minio-metrics-and-alerts>` ``minio_cluster_health_nodes_offline_count`` or :ref:`v2 <minio-metrics-v2>` ``minio_cluster_nodes_offline_total`` metrics to detect whether one or more MinIO nodes are offline.

Reviewer: For both the v3 and v2 metrics, I'd suggest adding an :ref: point at the `/cluster/heath` heading so that you can send users right to that location in the file rather than dumping them to the top of the page. Otherwise they have to really search hard to find the relevant metric.

Author: Linked to the existing refs for cluster metrics sections (both 2 & 3), although this is only a marginal improvement over the top of the page.

Anything more would have to be in the source readme, so for the moment we are stuck with that. The current automation can only do so much without getting really ugly.
